packages/backend/migrations/20230918113102_rename_payment_pointer_tables.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return Promise.all([
+    knex.schema.renameTable('paymentPointers', 'walletAddresses'),
+    knex.schema.renameTable('paymentPointerKeys', 'walletAddressKeys'),
+    knex.schema.alterTable('walletAddressKeys', function (table) {
+      table.renameColumn('paymentPointerId', 'walletAddressId')
+      table.foreign('walletAddressId').references('walletAddresses.id')
+    }),
+    knex.schema.alterTable('quotes', function (table) {
+      table.renameColumn('paymentPointerId', 'walletAddressId')
+      table.foreign('walletAddressId').references('walletAddresses.id')
+      table.index(['walletAddressId', 'createdAt', 'id'])
+    }),
+    knex.schema.alterTable('incomingPayments', function (table) {
+      table.renameColumn('paymentPointerId', 'walletAddressId')
+      table.foreign('walletAddressId').references('walletAddresses.id')
+      table.index(['walletAddressId', 'createdAt', 'id'])
+    }),
+    knex.schema.alterTable('outgoingPayments', function (table) {
+      table.renameColumn('paymentPointerId', 'walletAddressId')
+      table.foreign('walletAddressId').references('walletAddresses.id')
+      table.index(['walletAddressId', 'createdAt', 'id'])
+    }),
+    knex('webhookEvents')
+      .update({
+        // renames paymentPointer keys (if any) to walletAddress in data json column
+        data: knex.raw(
+          "data::jsonb - 'paymentPointer' || jsonb_build_object('walletAddress', data::jsonb->'paymentPointer')"
+        )
+      })
+      .whereRaw("data->'paymentPointer' is not null"),
+    knex('webhookEvents')
+      .update({
+        // renames paymentPointerId keys (if any) to walletAddressId in data json column
+        data: knex.raw(
+          "data::jsonb - 'paymentPointerId' || jsonb_build_object('walletAddressId', data::jsonb->'paymentPointerId')"
+        )
+      })
+      .whereRaw("data->'paymentPointerId' is not null"),
+    knex('webhookEvents')
+      .update({
+        // renames paymentPointerUrl keys (if any) to walletAddressUrl in data json column
+        data: knex.raw(
+          "data::jsonb - 'paymentPointerUrl' || jsonb_build_object('walletAddressUrl', data::jsonb->'paymentPointerUrl')"
+        )
+      })
+      .whereRaw("data->'paymentPointerUrl' is not null"),
+    knex('webhookEvents')
+      .update({
+        // renames payment_pointer.not_found values (if any) to wallet_address.not_found for type key in data json column
+        type: knex.raw("REPLACE(type, 'payment_pointer.', 'wallet_address.')")
+      })
+      .whereLike('type', 'payment_pointer%')
+  ])
```

Comment:
i inserted some dummy data before this migration and this and similar all seem to be working correctly